packages/react-magma-dom/src/components/NativeSelect/NativeSelect.tsx

```diff
@@ -100,40 +134,77 @@ export const NativeSelect = React.forwardRef<HTMLDivElement, NativeSelectProps>(
 
     const id = useGenerateId(defaultId);
 
+    // If the labelPosition is set to 'left' then a <div> wraps the FormFieldContainer, NativeSelectWrapper, and NativeSelect for proper styling alignment.
+    function AdditionalContentWrapper(props) {
+      if (
+        labelPosition === LabelPosition.left ||
+        (!labelText && labelPosition === LabelPosition.top)
+      ) {
+        return (
+          <StyledAdditionalContentWrapper theme={theme}>
+            {props.children}
+          </StyledAdditionalContentWrapper>
+        );
+      }
+      return props.children;
+    }
+
+    const hasAdditionalContent = additionalContent ? true : false;
+    const hasLabel = labelText ? true : false;
```

Comment:
You can try the double bang operator: https://dev.to/sanchithasr/what-is-the-double-bang-operator-in-javascript-4i3h